src/components/Select/Select.tsx

```diff
@@ -85,18 +95,22 @@ export default class Select extends React.Component<
 
         return (
             <select
+                name={name}
                 {...selectProps}
```

Comment:
I know this wasn't your work, but since you're here anyway I've been trying to move away from this and pass in props explicitly (since it's easier to read) - make that change here?

Reply:
So remove `...selectedProps` and put each prop individually?

Reply:
yup!

Reply:
Done.